### PR TITLE
Fix some clippy warnings

### PR DESCRIPTION
**Pull request by remilauzier**

### CODE REVIEW

File: src/render/debug.rs [hunk 1]
```diff
@@ -174,7 +174,7 @@ impl Context {
                 binding: 0,
                 resource: wgpu::BindingResource::Buffer(wgpu::BufferBinding {
                     buffer: &locals_buf,
-                    offset: 1 * locals_size,
+                    offset: locals_size,
```
Comment (kvark) [inline src/render/debug.rs:177]: this is intentional, let's tell clippy to leave it be

File: src/level/mod.rs [hunk 2]
```diff
@@ -16,7 +16,7 @@ pub type Altitude = u8;
 pub type Delta = Altitude;
 pub const DOUBLE_LEVEL: u8 = 1 << 6;
 pub const DELTA_SHIFT0: u8 = 2 + 3;
-pub const DELTA_SHIFT1: u8 = 0 + 3;
```
Comment (kvark) [inline src/level/mod.rs:19]: this is intentional

File: src/level/mod.rs [hunk 3]
```diff
@@ -300,17 +300,16 @@ impl LevelData {
                 if delta != 0 {
                     // average between two texels
                     let mat = avg(color[3], color[7]);
-                    level.meta[i + 0] =
```
Comment (kvark) [inline src/level/mod.rs:303]: these are all intentional
let's have a crate-level lint allowance for this rule

File: src/render/body.rs [hunk 4]
```diff
@@ -685,7 +685,7 @@ impl GpuStore {
                         buf_set_control.as_ref().unwrap(),
                         (index * size) as wgpu::BufferAddress,
                         &self.buf_data,
-                        (body_id * data_size + 0) as wgpu::BufferAddress,
+                        (body_id * data_size) as wgpu::BufferAddress,
```
Comment (kvark) [inline src/render/body.rs:688]: same here - intentional

**Review by kvark — COMMENTED** [review]
Thank you for contributing!

**Comment by remilauzier** [comment]
@kvark That should fix the problem.

File: src/level/mod.rs (hunk 2 above)
Comment (kvark) [reply src/level/mod.rs:19]: still a problem

File: src/level/mod.rs (hunk 3 above)
Comment (kvark) [reply src/level/mod.rs:303]: still a problem

File: src/render/debug.rs (hunk 1 above)
Comment (kvark) [reply src/render/debug.rs:177]: still a problem

**Comment by remilauzier** [comment]
@kvark This time that should be good.

**Review by kvark — APPROVED** [review]
thank you!

**Pull request merged**